ReceiveMessage/index.ts

```diff
@@ -17,33 +19,60 @@ const httpTrigger: AzureFunction = async function (context: Context, req: HttpRe
     const curUserState = await readUserRequest(req);
     context.log(curUserState);
     context.log(sentMessage);
-    const response = await formResponse(curUserState, sentMessage.Body);
+    // const response = await formResponse(curUserState, sentMessage.Body);
+    const response = await manageKeywordSent(sentMessage, curUserState);
 
     const message = new MessagingResponse();
     message.message(response);
 
     // if there's a conditional (like not recording all messages), put that here
     storeMessage(sentMessage, curUserState.currMessage);
-    
+
     context.res = {
         // status: 200, /* Defaults to 200 */ /*
         body: message.toString(),
         headers: { 'Content-Type': 'application/xml' },
         isRaw: true,
     };
-    
+
     context.done();
 };
 
-const storeMessage = async function (sentMessage:qs.ParsedQs, curMessageID: Schema.Types.ObjectId) {
-    const userMessage  = new MessageResponse({
+const storeMessage = async function (sentMessage: qs.ParsedQs, curMessageID: Schema.Types.ObjectId) {
+    const userMessage = new MessageResponse({
         accountID: sentMessage.From,
         chatBotMessageID: curMessageID,
-        response: sentMessage.Body});
+        response: sentMessage.Body,
+    });
     userMessage.save(function (err, mes) {
         if (err) return console.error(err);
-        console.log("Saved message to database");
+        console.log('Saved message to database');
     });
-}
+};
+
+//checks if a special keyword is in the message sent
+const manageKeywordSent = async function (sentMessage: qs.ParsedQs, curUserState) {
```

Comment:
This is really good stuff; based on the screenshots from Serena's messages we have all of the correct functionality done! 

However, this function isn't very sustainable or friendly to read, because if we want to keep adding more keywords to the app, we have to check for it here as well as add it to the specialMessageIds.ts Map. The solution to this is to convert the Map into a `<string, function>` map where the function returns the responseString we're looking for and handles any special cases. See the comments below on how we can go about doing that.